spec/features/shelters/shelters_index_spec.rb

```diff
@@ -93,4 +93,31 @@
     click_link("All Shelters")
     expect(page).to have_current_path("/shelters")
   end
+
+  it "has a link to re-order shelters alphabetically" do
+    shelter_1 = Shelter.create(name: "Denver Animal Shelter",
+                         address: "500 Invisible St.",
+                         city: "Denver",
+                         state: "Colorado",
+                         zip: "80201")
+
+    shelter_2 = Shelter.create(name: "Downtown Animal Shelter",
+                         address: "2550 WeWatta St.",
+                         city: "Denver",
+                         state: "Colorado",
+                         zip: "80222")
+
+    shelter_3 = Shelter.create(name: "Aurora Animal Shelter",
+                         address: "3665 E. Colfax Ave.",
+                         city: "Aurora",
+                         state: "Colorado",
+                         zip: "80399")
+
+    visit "/shelters"
+    expect(page).to have_link("Sort by Alphabetical Order")
+    click_link("Sort by Alphabetical Order")
+    expect(page).to have_current_path("/shelters?alpha=true")
+    first('.shelter_link').click_link
```

Comment:
excited to figure out a way to test sorting in capybara before checking it in chrome